# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by scripts/Create-CopyPatch.
# 
# T2 SDE: package/*/xemacs/hotfix-texinfo-6.7.patch
# Copyright (C) 2021 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- xemacs-21.5.34/man/lispref/lispref.texi.vanilla	2021-07-14 18:39:17.968923084 +0200
+++ xemacs-21.5.34/man/lispref/lispref.texi	2021-07-14 18:40:17.779919519 +0200
@@ -1,4 +1,6 @@
 \input texinfo  @c -*-texinfo-*-
+@documentencoding ISO-8859-1
+
 @c %**start of header
 @setfilename ../../info/lispref.info
 @c @smallbook
--- xemacs-21.5.34/man/xemacs-faq.texi.vanilla	2021-07-14 18:40:41.471918107 +0200
+++ xemacs-21.5.34/man/xemacs-faq.texi	2021-07-14 18:40:46.932917781 +0200
@@ -1,4 +1,6 @@
 \input texinfo.tex      @c -*- mode: texinfo; coding: iso-2022-8 -*-
+@documentencoding ISO-8859-1
+
 @c %**start of header
 @setfilename ../info/xemacs-faq.info
 @settitle Frequently asked questions about XEmacs
